extensions/msal-node-extensions/test/performance-test/LockAndWriteToStorageScript.js

```diff
@@ -7,7 +7,7 @@ const process = require("process");
 const extensions = require("../../dist/index");
 const fs = require("fs");
 const path = require("path");
-const msalCommon = require("@azure/msal-common");
+const msalCommon = require("../../dist/lib/msal-common/src/index");
```

Comment:
Curious about this. Do we have to import like this because this is a js file (rather than ts)?

Reply:
I see we're importing node-extensions the same way above, so that makes sense to me

Reply:
Yes, exactly.